docs/public/sandbox.js

```diff
@@ -0,0 +1,19 @@
+import * as R from "https://esm.sh/remeda";
+
+window.R = R;
+
+const sansSerif = "font-family: sans-serif; font-size: 16px";
+const monospace = "font-family: monospace; font-size: 16px";
+
+console.log(
+  "%cYou can use Remeda with the %cR%c global.",
+  sansSerif,
+  monospace,
+  sansSerif,
+);
+console.log(
+  "%cTry typing %cR.add(5, 10)%c here:",
+  sansSerif,
+  monospace,
+  sansSerif,
```

Comment:
Perhaps italics for the example?

Reply:
@eranhirsch 
<img width="497" alt="image" src="https://github.com/remeda/remeda/assets/15047511/56a80d8d-2a6c-4ffe-af26-06c609d8c515">
I thought so too, but monospace Italic looks a little weird, imho